desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/Breadcrumb.scss

```diff
@@ -0,0 +1,23 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.hue-path-browser__breadcrumb {
+  display: contents;
+
+  .hue-path-browser__overflowing-label {
```

Comment:
You shouldn't need nesting when using BEM, since BEM itself contains the structure.

Reply:
The nesting is because I want the min-width to be applied to the label only if it is a breadcrumb and not a menu item.

Reply:
Ok, this is not super important, but but I would let the breadcrumb component pass in a custom class to the overflowing component called .hue-path-browser__breadcrumb-overflow-label